src/http-gateways/trustless-gateway.md

```diff
@@ -235,6 +235,25 @@ In case both are present in the request, the value from the [`Accept`](#accept-r
 
 Below MUST be implemented **in addition** to "HTTP Response" of :cite[path-gateway].
 
+## Response Status Codes
+
+### `200` OK
+
+The request succeeded.
+
+If the HTTP method was `GET`, then data is transmitted in the message body.
+
+### `404` Not Found
+
+Error to indicate that requested resource is not provided by the gateway.
+
+### `3XX` Redirects
+
+Non-recursive trustless gateway SHOULD NOT return HTTP redirects.
+Change of URL should occur by announcing it to the routing system.
```

Comment:
> Change of URL should occur by announcing it to the routing system.

i'm slightly confused by this line..  which routing system? 

do you mean to imply that clients receiving a 3xx redirect from a spec-compliant trustless gateway should detect the 3xx and handle it appropriately?

maybe it's just too late for me.